### DISCUSSION TITLE
Stats that we want to track

### DISCUSSION THREAD

**Discussion by OBWANDO** [o3de/sig-operations#9] [Ideas]
Starting up a thread for us to put together things that we want to track from Discord.

**Comment by JustFinchy** [comment]
Activity on a week by week basis. 

Activity during news/updates released (To see how many people are active during any new information)

Busiest Sections on the discord. 

Questions/issues that are asked the most to be able to do a FAQ section based on it. 

Questions/issues that are asked, but never responded to

Questions/issues that are asked multiple times by the same user

Suggestions/requests made the most by users (For example, if 40 people are asking for more apple pie in the selection)

**Reply by Pshaowang** [reply]
@aFinchy, great list of metrics that we can start tracking. Below is my list (some overlap, but on a more granular level).

# Discord Metrics to Track
## Total O3DE server member count
* Daily count
* Weekly count
* Monthly count
## O3DE member count by server channels
* Daily count
* Weekly count
* Monthly count
## Total O3DE server message count
* Daily count
* Weekly count
* Monthly count
## O3DE channel message count
* Daily count
* Weekly count
* Monthly count
## Total O3DE server member status (in hours)
* Online
* Dnd (do not disturb)
* Idle
* Offline
## Top 10 trending topics on O3DE server broken down by
* Server channels – this is the 10 most popular topic/mentions on the 
server but labeled with the channel name they appear in.
* On a daily basis
